flixel/input/gamepad/XboxButtonID.hx

```diff
@@ -125,16 +125,16 @@ class XboxButtonID
 	/**
 	 * Button IDs
 	 */
-	public static inline var A:Int = 10;
-	public static inline var B:Int = 11;
-	public static inline var X:Int = 12;
-	public static inline var Y:Int = 13;
-	public static inline var LB:Int = 8;
-	public static inline var RB:Int = 9;
-	public static inline var BACK:Int = 5;
-	public static inline var START:Int = 4;
-	public static inline var LEFT_ANALOG:Int = 6;
-	public static inline var RIGHT_ANALOG:Int = 7;
+	public static inline var A:Int = 0;
+	public static inline var B:Int = 1;
+	public static inline var X:Int = 2;
+	public static inline var Y:Int = 3;
+	public static inline var LB:Int = 4;
+	public static inline var RB:Int = 5;
+	public static inline var BACK:Int = 6;
+	public static inline var START:Int = 7;
+	public static inline var LEFT_ANALOG:Int = 8;
+	public static inline var RIGHT_ANALOG:Int = 9;
 	
 	public static inline var XBOX:Int = 14;
 	
```

Comment:
Yeah I think ultimately we should do that. There's other things to reconcile too... on some controller models the analog triggers have digital buttons at the end that "click", on some controllers (ps3/ps4) the face buttons have analog pressure... it's a lot to wrap, but the basics should be doable without exploding the scope of this PR.

Just for now I think generating fake button events based on hat stuff in legacy will work for now, and then we can map more cleanly when targeting next.